core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/DataAccessImpl.java

```diff
@@ -1053,16 +1057,25 @@ private BoundStatement bindDataPoint(PreparedStatement statement, Metric<?> metr
 
         // Max time back is <24 hours.
         if(startZone.isBefore(endZone.minus(23, ChronoUnit.HOURS))) {
+            log.infof("Adjusting the startZone\n");
```

Comment:
I would be inclined to change some of these info level log statements to debug or trace.